### ISSUE TITLE
aurelia-templating.d.ts exports need update

### ISSUE THREAD

**Issue by spi-ap** [aurelia/templating#212]
Running the latest update, I get the following TypeScript errors.
```
xxx\github\aurelia\templating@0.17.5\aurelia-templating.d.ts(8,21): error TS2305: Module ''aurelia-binding'' has no exported member 'createOverrideContext'.
xxx\github\aurelia\templating@0.17.5\aurelia-templating.d.ts(8,44): error TS2305: Module ''aurelia-binding'' has no exported member 'ValueConverterResource'.
xxx\github\aurelia\templating@0.17.5\aurelia-templating.d.ts(8,68): error TS2305: Module ''aurelia-binding'' has no exported member 'BindingBehaviorResource'.
xxx\github\aurelia\templating@0.17.5\aurelia-templating.d.ts(8,93): error TS2305: Module ''aurelia-binding'' has no exported member 'subscriberCollection'.
xxx\github\aurelia\templating@0.17.5\aurelia-templating.d.ts(8,145): error TS2305: Module ''aurelia-binding'' has no exported member 'EventManager'.
xxx\github\aurelia\templating@0.17.5\aurelia-templating.d.ts(8,159): error TS2305: Module ''aurelia-binding'' has no exported member 'createScopeForTest'.
```

**Comment by jdanyow** [comment]
`jspm update` will fix this (aurelia-binding v0.11.4)

**Comment by spi-ap** [comment]
I already had aurelia-binding 0.11.4.
```
     Installed versions of github:aurelia/binding
       aurelia-binding 0.11.4 (^0.11.0)
```
